modules/configuration/src/main/java/org/apache/ignite/internal/configuration/tree/NamedListNode.java

```diff
@@ -484,6 +499,19 @@ public NamedListNode<N> copy() {
         return new NamedListNode<>(this);
     }
 
+    private void assertMutability() {
```

Comment:
Missing javaDoc, why not `final` ?

Reply:
But this method is important!

Reply:
Anyway, why would I make private method "final", it's already effectively final